hail/Makefile

```diff
@@ -34,6 +34,8 @@ JAVAC ?= javac
 JAR ?= jar
 endif
 
+ELASTIC_MAJOR_VERSION ?= 6
```

Comment:
You should move this up into the hail/spark/scala version block, and add a call to `ENV_VAR` like on line 20.